packaging/installer/methods/kickstart.md

```diff
@@ -11,92 +11,133 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
+normal user:
 
 ```bash
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --install /usr/local/
+wget -O ./kickstart.sh https://my-netdata.io/kickstart.sh && sh ./kickstart.sh
 ```
 
 > See our [installation guide](/packaging/installer/README.md) for details about [automatic updates](/packaging/installer/README.md#automatic-updates) or
 > [nightly vs. stable releases](/packaging/installer/README.md#nightly-vs-stable-releases).
 
+> If you are installing on macOS, make sure to check the [install documentation for macOS](packaging/installer/methods/macos) before continuing.
+
 ## What does `kickstart.sh` do?
 
-The `kickstart.sh` script does the following after being downloaded and run using `bash`:
+The `kickstart.sh` script does the following after being downloaded and run using `sh`:
 
--   Detects the Linux distribution and **installs the required system packages** for building Netdata. Unless you added
-    the `--dont-wait` option, it will ask for your permission first.
--   Checks for an existing installation, and if found updates that instead of creating a new install.
--   Downloads the latest Netdata source tree to `/usr/src/netdata.git`.
--   Installs Netdata by running `./netdata-installer.sh` from the source tree, using any [optional
-    parameters](#optional-parameters-to-alter-your-installation) you have specified.
--   Installs `netdata-updater.sh` to `cron.daily`, so your Netdata installation will be updated with new nightly
-    versions, unless you override that with an [optional parameter](#optional-parameters-to-alter-your-installation).
--   Prints a message whether installation succeeded or failed for QA purposes.
+- Determines what platform you are running on.
+- Checks for an existing installation, and if found updates that instead of creating a new install.
+- Attempts to install Netdata using our [official native binary packages](#native-packages).
+- If there are no official native binary packages for your system (or installing that way failed), tries to install
+  using a [static build of Netdata](#static-builds) if one is available.
+- If no static build is available, installs required dependencies and then attempts to install by 
+  [building Netdata locally](#local-builds) (by downloading the sources and building them directly).
+- Installs `netdata-updater.sh` to `cron.daily`, so your Netdata installation will be updated with new nightly
+  versions, unless you override that with an [optional parameter](#optional-parameters-to-alter-your-installation).
+- Prints a message whether installation succeeded or failed for QA purposes.
 
 ## Optional parameters to alter your installation
 
-The `kickstart.sh` script passes all its parameters to `netdata-installer.sh`, which you can use to customize your
-installation. Here are a few important parameters:
+The `kickstart.sh` script accepts a number of optional parameters to control how the installation process works:
```

Comment:
Maybe it should be noted that a combination of the kickstart.sh parameters is not allowed.

Reply:
@cpipilas In the sense of: 

> Note: You can only use one parameter when you run the kickstart script.

Reply:
I’m a bit confused by this myself, as there isn’t any such limitation that I know of (and the new kickstart code gives useful error messages when incompatible options are specified, unlike the old one).